source/fem-dem/cfd_dem_coupling.cc

```diff
@@ -1267,7 +1267,22 @@ CFDDEMSolver<dim>::print_particles_summary()
         << "--------------------------------------------------------------------------"
         << "--------------------------------------------------------------------------"
         << std::endl;
+
+      particle_velocity_table.add_value(
```

Comment:
I would use a separate member and use a parameter to do this. I think it would be cleaner this way :)

Reply:
I will remove it. It was only needed for one comparison case